pkg/manager/runtime_env/handler.go

```diff
@@ -12,10 +12,41 @@ import (
 	"google.golang.org/grpc/status"
 
 	"openpitrix.io/openpitrix/pkg/pb"
+	"openpitrix.io/openpitrix/pkg/utils/sender"
+	"openpitrix.io/openpitrix/pkg/logger"
 )
 
 func (p *Server) CreateRuntimeEnv(ctx context.Context, req *pb.CreateRuntimeEnvRequest) (*pb.CreateRuntimeEnvResponse, error) {
-	return nil, status.Errorf(codes.Internal, "CreateRuntimeEnv: %+v", fmt.Errorf("hello world"))
+	//TODO: validation request
+	s := sender.GetSenderFromContext(ctx)
+	logger.Infof("Got sender: %+v", s)
```

Comment:
These two lines can be deleted, and then we will add the general grpc debug log.